docs/CONTRIBUTING.md

```diff
@@ -62,15 +90,39 @@ Example, should we add `fasttemplate`
 
 No, we should not add that dependency.
 
-### Test Policy
+#### Test Policy
 
 Changes without either unit or e2e tests are unlikely to be accepted.
 See [the pull request template](https://github.com/argoproj/argo-workflows/blob/main/.github/pull_request_template.md).
 
-### Contributor Workshop
+### Other Contributions
 
-Please check out the following resources if you are interested in contributing:
+* [Reviewing PRs](#reviewing-prs)
+* Responding to questions in the [Slack](#slack) channels
+* Responding to questions in [Github Discussions](https://github.com/argoproj/argo-workflows/discussions)
+* [Triaging new bugs](#triaging-bugs)
 
-* [90m hands-on contributor workshop](https://youtu.be/zZv0lNCDG9w).
-* [Deep-dive into components and hands-on experiments](https://docs.google.com/presentation/d/1IU0a3unnr3tBRi38Zn3EHQZj3z6yvocfG9x9icRu1LE/edit?usp=sharing).
-* [Architecture overview](https://github.com/argoproj/argo-workflows/blob/main/docs/architecture.md).
+#### Reviewing PRs
+
+Anybody can review a PR.
+If you are in a [designated role](#roles), add yourself as an "Assignee" to a PR if you plan to lead the review.
+If you are a Reviewer or below, then once you have approved a PR, request a review from one or more Approvers and above.
+
+#### Triaging Bugs
+
+New bugs need to be triaged to identify the highest priority ones.
+
+Apply the labels `P0`, `P1`, `P2`, and `P3`, where `P0` is highest priority and needs immediate attention, followed by `P1`, `P2`, and then `P3`.
+If there's a new `P0` bug, notify the [#argo-wf-contributors](https://cloud-native.slack.com/archives/C0510EUH90V) slack channel.
```

Comment:
```suggestion
If there's a new `P0` bug, notify the [`#argo-wf-contributors`](https://cloud-native.slack.com/archives/C0510EUH90V) Slack channel.
```

For consistency with the "Slack" section above. This will also make the `.spelling` addition no longer needed